scripts/disabled_tests/tests/test_playlist_parser.py

```diff
@@ -0,0 +1,18 @@
+import json
+import os
+from unittest import TestCase
+
+from playlist_parser import parse_all_files
```

Comment:
Not sure if I missed anything, when I run python scripts/disabled_tests/tests/test_playlist_parser.py, it gave error: 
```
Traceback (most recent call last):
  File "scripts/disabled_tests/tests/test_playlist_parser.py", line 5, in <module>
    from playlist_parser import parse_all_files
ModuleNotFoundError: No module named 'playlist_parser'

Reply:
Hmm true the `README` is lacking in terms of how to run the unit tests.

See instructions added to README in 8ff24151ee40335c94c8dc0441509013b2420228